src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+.red {
+  background: red;
+}
+
+.green {
+  background: green;
+}
+
+.blue {
+  background: blue;
+}
+
+.myHeight {
```

Comment:
height-100 - use this class name,
`myHeight` is an abstract name, which is not good for class names